src/main/java/org/gentoo/java/ebuilder/portage/MavenVersion.java

```diff
@@ -14,7 +14,7 @@
      * Pattern for parsing maven version number.
      */
     private static final Pattern PATTERN_VERSION = Pattern.compile(
-            "^(\\d+)(?:\\.(\\d+))?(?:\\.(\\d+))?(?:-(.*))?$");
+            "^(\\d+)(?:\\.(\\d+))?(?:[\\.b](\\d+))?(?:[\\.-](.*))?$");
```

Comment:
The present approach is that, treating "b" as ".", so that the part after "b" becomes an incremental version.   .GA will be identical to -GA.

> i think this should probably also affect resolution of versioned packages from the cache.

I think not, as both the cache and dependency section of pom.xml share the same MavenVersion parser.

A comment in the code is a good idea.